docs/guide.rst

```diff
@@ -298,16 +298,16 @@ You can make requests using the `httpie <https://httpie.org/>`_ utility.
         "data": [
             {
                 "author_id": 2,
-                "created_at": "2019-06-16T01:09:33.450768+00:00",
+                "created_at": "2019-06-16T01:09:33.450768",
```

Comment:
Ick.

Maybe I should add a note that these are only naive because it's SQLite on the back end here?

Reply:
did that